### ISSUE TITLE
registerAllFlagsListener returns bizarre data in string response

### ISSUE THREAD

**Issue by nealchandra** [launchdarkly/react-native-client-sdk#61]
**Describe the bug**
When the change listener for `registerAllFlagsListener` is called, it seems to pass a string value with a lot of additional data other than simply the names of changed feature flags. Here is an example of the response I get:

```
["enableMobileTestFlag": LaunchDarkly.LDChangedFlag(key: "enableMobileTestFlag", oldValue: Optional(1), oldValueSource: Optional(LaunchDarkly.LDFlagValueSource.server), newValue: Optional(0), newValueSource: Optional(LaunchDarkly.LDFlagValueSource.server))]
```

This whole value is a string response, and hard to parse on the JS side as it isn't even valid JSON. It seems like perhaps the native side representation of this data changed, and what is now getting passed back is essentially gibberish on the JS side.
 
**To reproduce**
Set up a change listener and log the response.

**Expected behavior**
I would expect the response to simply have the shape:

```
['enableMobileTestFlag']
```

Or, even more usefully, if the callback was passed the updated values rather than just the keys that have changed, that would be a lot more useful to us.

**Logs**
If applicable, add any log output related to your problem.

**SDK version**
3.2.1, the latest

**Language version, developer tools**
For instance, Go 1.11 or Ruby 2.5.3. If you are using a language that requires a separate compiler, such as C, please include the name and version of the compiler too.

**OS/platform**
For instance, Ubuntu 16.04, Windows 10, or Android 4.0.3. If your code is running in a browser, please also include the browser type and version.

**Additional context**

**Comment by torchhound** [comment]
Hi @nealchandra, thanks for reporting this.

_Filed internally as 95404_

**Comment by torchhound** [comment]
Hi @nealchandra `registerAllFlagsListener` seems to work correctly on Android but I've been able to reproduce this on iOS. A fix should be coming soon, thanks again for reporting this

**Comment by torchhound** [comment]
Hi @nealchandra this bug is fixed in version 3.2.2.

**Comment by nealchandra** [comment]
Thank you for integrating the fix quickly! We will upgrade and try it out this week :)
